documentation/registration.md

```diff
@@ -1,8 +1,37 @@
-# Registration
+# Registration
+
 ---
-## Registration with password
+
+## Registration with password
 
 1. Display the registration UI to the user in the app.
   
 2. When the user submits the registration form, call `registerWithPassword` on the client, passing the user password and map of traits.
-  
\ No newline at end of file
+
+## Registration with passkey
+
+1. Display the registration UI to the user in the app.
+2. When the user submits the registration form, call `registerWithProfile` first, it will return a 400 error with the passkey challenge.
```

Comment:
The `400 error` is irrelevant here. The library handles the status codes directly.

Reply:
You are right, fixed